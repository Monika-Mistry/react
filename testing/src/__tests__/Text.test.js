import React from 'react';
import { shallow , mount} from 'enzyme';
import renderer from 'react-test-renderer'

import { Text } from '../Text';

it('renders a text', () => {
    const wrapper = renderer.create(<Text />).toJSON();
    expect(wrapper).toMatchSnapshot();
})

