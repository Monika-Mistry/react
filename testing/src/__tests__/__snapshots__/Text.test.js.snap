// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a text 1`] = `
<div>
  <p>
    Text Component
  </p>
  <p>
    Some more content
  </p>
  <p>
    Content
  </p>
  <p>
    ksfcbhksdfsdffssdfsdfsajawebk
  </p>
</div>
`;
